config/policies.js

```diff
@@ -33,44 +37,44 @@ module.exports.policies = {
   },
 
   'game/reconnect': ['isLoggedIn', 'isInGame'],
-  'game/create': ['isLoggedIn', 'hasGameName'],
+  'game/create': [conditionalIsLoggedIn, 'hasGameName'],
```

Comment:
Could possibly try a different adjustment by adding an opponent specific routes for mock testing that uses the normal route and doesn't have a logged in check.